### PR TITLE
index.html wording fixes

### LINKED ISSUE #2: Change the title when we share the link
image

### PR DESCRIPTION
**Pull request by waf**
Some very minor fixes:

1. The text on the webpage says that event is over, which could hurt attendance. I've replaced it with a better message I found from another [.NET Conf 2018 group](https://www.meetup.com/MississaugaNETUG/events/255278028/).
    - Just FYI, the description on eventbrite says this too.
1. I noticed when the link is shared on WhatsApp it's described as "Evento-Event Html Template". It must be looking at the meta description tag. This might also happen if it's shared on Facebook and other sites.

Fixes #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -240,12 +240,10 @@ <h3 class="title">
...
-                        .NET Conf 2018 is a wrap!
-                        A huge THANK YOU to all who tuned in live, asked questions, and participated in our twitter
-                        feed. We had a lot of awesome sessions from various product teams and community experts that
-                        showed us all sorts of cool things we can build with .NET across platforms and devices. There’s
-                        never been a better time to be a .NET developer! Just getting started with .NET? Head to
-                        dot.net.
+                        Join us for .NET Conf, an event where we will be showcasing many of the advancements and
+                        capabilities of .NET. including some of the cool technologies that were recently announced by Microsoft.
+
+                        These technologies (but not limited to) include Web, Mobile Desktop, .NET Core, EF Core, C#, IoT, AI, ML, and Mobile.
```
Comment (aaronamm) [inline index.html]: Thank you for your contribute on this information, I really appreciate it.

File: index.html [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
     <meta charset="UTF-8">
-    <meta name="description" content="Evento -Event Html Template">
-    <meta name="keywords" content="Evento , Event , Html, Template">
+    <meta name="description" content="Thailand .NET Conf 2018">
+    <meta name="keywords" content="dotnet, .NET, conference, Thailand">
     <meta name="author" content="ColorLib">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <!-- ========== Title ========== -->
-    <title> dotnet conf - Thailand</title>
+    <title>dotnet conf - Thailand</title>
```
Comment (jerriep) [inline index.html]: Change `dotnet conf` to `.NET Conf` for consistency (and that is also how MS writes it)

**Comment by waf** [comment]
@aaronamm done. I put the `dotnet conf` title fix in a separate commit, because I recognize that change could be personal preference so we can back it out if needed.

Fixes #2

**Comment by kidchenko** [comment]
I added other improvements and I am merging this PR, thank you all <3, you can see the last changes live in the site.

**Pull request merged**